src/Catty/PlayerEngine/Scene/CBScene.swift

```diff
@@ -49,6 +50,7 @@ final class CBScene: SKScene {
         self.broadcastHandler = broadcastHandler
         self.formulaManager = formulaManager
         self.soundEngine = soundEngine
+        self.frameCounter = PlayerConfig.NumberOfFramesPerSpriteNodeUpdate
```

Comment:
Should that not be zero in order to execute the `update` function on every second frame? Wouldn't that mean that the function is called on the first run already?